src/Dofs/sparsity_pattern.jl

```diff
@@ -309,6 +309,10 @@ for (func,                              pre_f,
                                 cell_dofs = celldofs(dh,cell_idx)[element_dof_start + 1 : element_dof_start + getnbasefunctions(fi)]
                                 neighbour_dof_start = 0
                                 for fi2 in fh.field_interpolations
+                                    if(!IsDiscontinuous(typeof(fi2)<: VectorizedInterpolation ? typeof(fi2.ip) : typeof(fi2)))
```

Comment:
I would like to propose to dispatch this in the interpolation api via something like `IsDiscontinuous(ip::VectorizedInterpolation) = IsDiscontinuous(ip.ip)`

Reply:
Done, thanks!
It's now dispatched for both variables and types, we can remove one if it can cause confusion.